addons/survey/views/survey_templates_statistics.xml

```diff
@@ -386,6 +388,11 @@
                                     <i t-if="question.is_scored_question and choice_data['suggested_answer'].is_correct" class="fa fa-check"/>
                                 </p>
                             </td>
+                            <td t-if="has_image">
+                               <div t-if="choice_data['suggested_answer'].value_image"
```

Comment:
Answering here as github will not let me answer to your answer.

It's fine to keep the t-if if we do not want to have the ugly placeholder, I did not think about that :)